climada/engine/unsequa/unc_output.py

```diff
@@ -741,9 +752,8 @@ def plot_sensitivity(self, salib_si='S1', salib_si_conf='S1_conf',
         figsize : tuple(int or float, int or float), optional
             The figsize argument of matplotlib.pyplot.subplots()
             The default is derived from the total number of plots (nplots) as:
-
-            >>> nrows, ncols = int(np.ceil(nplots / 3)), min(nplots, 3)
-            >>> figsize = (ncols * FIG_W, nrows * FIG_H)
+                nrows, ncols = int(np.ceil(nplots / 3)), min(nplots, 3)
+                figsize = (ncols * FIG_W, nrows * FIG_H)
```

Comment:
Style?

Reply:
I don't know why parts of the code changed that I didn't mean to change. I reverted these parts back to what is in develop. This answer also applies to the next comments.